src/Controls/tests/Xaml.UnitTests/AssemblyInfoTests.cs

```diff
@@ -43,24 +43,27 @@ public void AssemblyVersion(string assemblyName)
 			Assert.AreEqual(0, actual.Revision, actual.ToString());
 		}
 
-		[Test, TestCaseSource(nameof(references))]
-		public void FileVersion(string assemblyName)
-		{
-			Assembly testAssembly = System.Reflection.Assembly.Load(assemblyName);
-			FileVersionInfo actual = FileVersionInfo.GetVersionInfo(testAssembly.Location);
-			string versionString = GetFileFromRoot(s_gitInfoFile);
-			if (versionString.IndexOf("-") is int idx && idx > 0)
-				versionString = versionString.Substring(0, idx);
-			Version expected = Version.Parse(versionString);
-			Assert.AreEqual(expected.Major, actual.FileMajorPart, $"FileMajorPart is wrong. {actual.ToString()}");
-			Assert.AreEqual(expected.Minor, actual.FileMinorPart, $"FileMinorPart is wrong. {actual.ToString()}");
-			// Fails locally
-			//Assert.AreEqual(expected.Build, actual.FileBuildPart, $"FileBuildPart is wrong. {actual.ToString()}");
-			//We need to enable this
-			//	Assert.AreEqual(ThisAssembly.Git.Commits, version.FilePrivatePart);
-			Assert.AreEqual(s_productName, actual.ProductName);
-			Assert.AreEqual(s_company, actual.CompanyName);
-		}
+		// [Test, TestCaseSource(nameof(references))]
+		// public void FileVersion(string assemblyName)
```

Comment:
Why was this disabled?

Reply:
By default the FileVersion is 42.42..42 on debug builds , so this testt needs to be rewrriten 
https://github.com/dotnet/arcade/blob/e543752a4d92d8df16a9f95b89fcfdc50aeabf74/Documentation/CorePackages/Versioning.md?plain=1#L238

Reply:
Should we rather wrap it in a `#if !DEBUG` since we don't really need this test in debug builds.